src/Modules/FreeModules-graded.jl

```diff
@@ -300,7 +300,8 @@ mutable struct BiModArray{T}
       r.SF = parent(s[1])
     end
     r.S = s
-    r.O = Array{FreeModuleElem_dec{S}, 1}(undef, Singular.ngens(s))
+    #r.O = Array{FreeModuleElem_dec{S}, 1}(undef, Singular.ngens(s))
+    r.O = [convert(F, s[i]) for i=1:Singular.ngens(s)]
```

Comment:
Shouldn't `convert(F, x)` be done via coercion `F(x)` these days? Perhaps this would work better?
```suggestion
    r.O = [F(x) for x in gens(s)]
```

Reply:
I think there is no coercion for Singular module elements.